tlscommon.c

```diff
@@ -535,13 +535,43 @@ void tlsreloadcrls() {
 	conf = (struct tls *)entry->data;
 #ifdef RADPROT_TLS
 	if (conf->tlsctx) {
+            if (conf->certkeypwd) {
+                debug(DBG_INFO, "tls: set default passwd for certkeypwd");
+                SSL_CTX_set_default_passwd_cb_userdata(conf->tlsctx, conf->certkeypwd);
+                SSL_CTX_set_default_passwd_cb(conf->tlsctx, pem_passwd_cb);
+            }
```

Comment:
Not much sense in redoing the password callback. It can't have changed.

Reply:
Thanks for your comments Fabian. Agree with you, this block is not needed.